### PR TITLE
test: bring up

### PR DESCRIPTION
**Pull request by SukkaW**

### CODE REVIEW

**Comment by curbengh** [comment]
After https://github.com/hexojs/hexo-filter-nofollow/pull/3, you should also create a test where plugin is disabled.

**Comment by SukkaW** [comment]
@curbengh I have no idea how to test disabling plugin.. disabling plugin is done in `index.js`, not in `lib/filter.js`

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+const should = require('chai').should(); // eslint-disable-line
```
Comment (curbengh) [inline test/index.js]: you can just `require('chai').shoud();` then you don't need eslint-disable and `globals` in eslintrc

File: package.json [hunk 2]
```diff
@@ -32,7 +32,10 @@
   },
   "homepage": "https://github.com/hexojs/hexo-filter-nofollow#readme",
   "devDependencies": {
+    "chai": "^4.2.0",
     "eslint": "^6.2.2",
-    "eslint-config-hexo": "^3.0.0"
+    "eslint-config-hexo": "^3.0.0",
+    "hexo": "github:hexojs/hexo",
```
Comment (curbengh) [inline package.json]: "hexojs/hexo"

**Comment by SukkaW** [comment]
@curbengh Is it ok to merge then?

**Pull request merged**